### DISCUSSION TITLE
Another discussion on Tor

### DISCUSSION THREAD

**Discussion by sirk3** [rdeepak2002/reddit-place-script-2022#341] [Q&A]
So  I want to set up the config file using tor instead of a proxy.

Since I don't have as much knowledge with it as many of you probably have. Could someone tell me how to set that part of the config up?

**Comment by ixi2101** [comment]
@LukachoLombardi

**Reply by LukachoLombardi** [reply]
on windows just set "using_tor" to true and make sure to enable "use_builtin_tor"

**Comment by Risewill23** [comment]
If your on windows you can just set `"using_tor": false,` to `"using_tor": true,`
